generator/uscore/metadata_extractor.rb

```diff
@@ -558,10 +548,36 @@ def add_special_cases(metadata)
         end
 
         unchanged_vital_sign_sequences.each do |sequence|
-          sequence[:must_supports][:elements].reject! { |el| el[:path].include?('component') }
+          sequence[:must_supports][:elements].delete_if do |element|
+            element[:path].include?('component')
+          end
+        end
+
+        # remove MS check for value[x] from Observation-bp
+        # Observation-bp profile uses Observation.component.value for Systolic and Diastolic pressures
+        bp_sequence = metadata[:sequences].find { |sequence| sequence[:profile] == 'http://hl7.org/fhir/StructureDefinition/bp' }
+        bp_sequence[:must_supports][:elements].delete_if do |element|
+          element[:path].start_with?('value[x]')
+        end
+        bp_sequence[:must_supports][:slices].delete_if do |slice|
+          slice[:path].start_with?('value[x]')
+        end
+
+        # ONC clarified that health IT developers that always provide HL7 FHIR "observation" values
+        # are not required to demonstrate Health IT Module support for "dataAbsentReason" elements.
+        # Remove MS check for dataAbsentReason and component.dataAbsentReason from vital sign profiles
```

Comment:
This is only for vital signs, not for all observations?

Reply:
Good catch! My first impression is that this is only for vital sign profiles. After reading CCG again, I think it means all Observation profiles.